desktop/index.md

```diff
@@ -49,6 +49,19 @@ Docker Desktop is available for Mac and Windows. For download information, syste
 
 For information about Docker Desktop licensing, see [Docker Desktop License Agreement](../subscription/index.md#docker-desktop-license-agreement).
 
+## Sign in to Docker Desktop
+
+After you’ve successfully installed and started Docker Desktop, we recommend that you authenticate using the **Sign in/Create ID** option from the Docker menu.
+
+Authenticated users get a higher pull rate limit compared to anonymous users. For example, if you are authenticated, you get 200 pulls per 6 hour period per IP address, compared to 100 pulls per 6 hour period per IP address for anonymous users. For more information, see [Download rate limit](../docker-hub/download-rate-limit.md).
+
+Authenticating through Docker Desktop also allows administrators to track the number of users who are using Docker Desktop. Additionally, through the [Image Access Management](../docker-hub/image-access-management.md) feature, administrators can configure Docker Desktop and set up guardrails to allow team members to:
```

Comment:
I know I read this once before, but now I'm wondering whether the first sentence of this is really true. Do admins have a way to count the number of users using DD? Even once we have SSO, will they?

Reply:
I thought it was possible to see how many users are running Docker Desktop when users sign in to specific organization (at least through the Docker Business subscription).

Reply:
I don't think that's true currently. CC @izzychu can you help confirm?